packages/components/app/styles/components/accordion.scss

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// ACCORDION
+//
+
+@use "../mixins/button" as *;
+@use "../mixins/focus-ring" as *;
+
+:root {
+  --token-accordion-item-padding: 16px;
+  --token-accordion-item-padding-right: 12px;
+}
+
+// ACCORDION COMPONENT (wrapper)
+
+.hds-accordion {
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+}
+
+// ACCORDION ITEM COMPONENT (nested child)
+
+.hds-accordion-item {
+  width: 100%;
```

Comment:
area you sure you need this? if I comment it out, I don't see differences 🤔

Reply:
Oh, I probably don't now. I'll remove, thanks.